framework/sandstone_opts.cpp

```diff
@@ -266,47 +264,45 @@ Common command-line options are:
+     single thread per test invocation. A value of less than or equal to 0 means
+     that there is no limit. The default value is 128. Sandstone will not log
+     partial data, so if the binary data would cause the thread to exceed this
+     threshold it simply will not be output.
  -n <NUMBER>, --threads=<NUMBER>
-     Set the number of threads to be run to <NUMBER>. If not specified or if
-     0 is passed, then the test defaults to the number of CPUs in the system.
+     Set the number of threads to be run to <NUMBER>. If not specified or if 0
+     is passed, then the test defaults to the number of CPUs in the system.
      Note the --cpuset and this parameter do not behave well together.
  -o, --output-log <FILE>
-     Place all logging information in <FILE>.  By default, a file name is
-     auto-generated by the program.  Use -o /dev/null to suppress creation of any file.
+     Place all logging information in <FILE>. By default, a file name is
+     auto-generated by the program. Use -o /dev/null to suppress creation of any
+     file.
  -s <STATE>, --rng-state=<STATE>
      Specify the random generator state to reload. The seed is in the form:
-       Engine:engine-specific-data
+     Engine:engine-specific-data
  -v, -q, --verbose, --quiet
-     Set logging output verbosity level.  Default is quiet.
+     Set logging output verbosity level. Default is quiet.
  --version
      Display program version information.
  --1sec, --30sec, --2min, --5min
-     Run for the specified amount of time in the option. In this mode, the program
-     prioritizes test execution based on prior detections.
-     These options are intended to drive coverage over multiple runs.
-     Test priority is ignored when running in combination with the
-     --test-list-file option.
+     Run for the specified amount of time in the option. In this mode, the
+     program prioritizes test execution based on prior detections. These
+     options are intended to drive coverage over multiple runs. Test priority
+     is ignored when running in combination with the --test-list-file option.
  --test-list-file <file path>
-     Specifies the tests to run in a text file.  This will run the tests
-     in the order they appear in the file and also allows you to vary the
-     individual test durations.  See the User Guide for details.
+     Specifies the tests to run in a text file. This will run the tests in the
+     order they appear in the file and also allows you to vary the individual
+     test durations. See the User Guide for details.
  --test-list-randomize
      Randomizes the order in which tests are executed.
  --test-delay <time in ms>
      Delay between individual test executions in milliseconds.
-  -Y, --yaml [<indentation>]
-     Use YAML for logging. The optional argument is the number of spaces to
-     indent each line by (defaults to 0).
-For more options and information, please see the User Reference
-Guide.
+  -Y, --yaml
```

Comment:
Aside from the fact that you're now no longer saying this takes an optional argument, this is ok.

`getopt` supports `-vY` because `-v` takes no options, but `-Yv` won't work.